docs/full/cli-architecture.md

```diff
@@ -172,7 +172,7 @@ Note: For backward compatibility reasons, one `mission-control` context can be a
 
 Target represents a group of Tanzu CLI Plugins that are of the same category (generally talking to similar endpoints) and are represented as the `tanzu <target-name> <plugin-name>` command.
   
-The Tanzu CLI supports three targets: `global`, `kubernetes` (alias `k8s`), and `mission-control` (alias `tmc`).
+The Tanzu CLI supports three targets: `global`, `kubernetes` (alias `k8s`), `mission-control` (alias `tmc`) and `operations` (alias `ops`).
```

Comment:
supports three targets -> supports four targets 😄

Reply:
Lol. Trying to get things too fast.